### ISSUE TITLE
Add Price for Lepton AI Models

### ISSUE THREAD

**Issue by bobmayuze** [AgentOps-AI/tokencost#12]
https://www.lepton.ai/pricing

bunch of models provided : llama7b,13b,70b, codellama 7b,13b,34b, mixtral-8*7b, etc...

**Comment by areibman** [comment]
> https://www.lepton.ai/playground
> 
> bunch of models provided : llama7b,13b,70b, codellama 7b,13b,34b, mixtral-8*7b, etc...
> 
> For pricing: https://www.lepton.ai/pricing

I'm assuming these are the prices:

Model | Price
-- | --
Llama2 7b | $0.1 / million tokens
Llama2 13b | $0.3 / million tokens
Llama2 70b | $0.8 / million tokens
Code Llama 7b | $0.1 / million tokens
Code Llama 13b | $0.3 / million tokens
Code Llama 34b | $0.8 / million tokens
AIQR | $0.00015 / step
Stable Diffusion XL | $0.00015 / step
Super Resolution | $0.00015 / image
WhisperX | $0.00007 / second

Do these models have canonical names? i.e. "gpt-3.5-turbo" = GPT-3.5. Looks like a simple thing to add

**Comment by bobmayuze** [comment]
yep, llama2-7b, llama2-13b, llama2-70b, mixtral-8x7b, codellama-7b, codellama-13b, codellama-34b

**Comment by bobmayuze** [comment]
and they are open AI compatible, eg. for mixtral-8*7b https://www.lepton.ai/playground/chat?model=mixtral-8x7b

**Comment by areibman** [comment]
> and they are open AI compatible, eg. for mixtral-8*7b https://www.lepton.ai/playground/chat?model=mixtral-8x7b

Sounds good. I can add these, but if you'd prefer, raise a PR and I'll approve.

**Comment by bobmayuze** [comment]
https://github.com/AgentOps-AI/tokencost/blob/main/tokencost/constants.py

**Comment by areibman** [comment]
> I'm assuming the structure of [this file](https://github.com/AgentOps-AI/tokencost/blob/main/tokencost/constants.py) needs to be altered a bit. maybe add a provider attribute to each model? and default provider to openAI? or have a list of providers as the price do vary among these providers.

This pattern from LiteLLM might work. Simply adding slashes:

https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json

**Comment by areibman** [comment]
@bobmayuze I've created a new cost dictionary file: https://github.com/AgentOps-AI/tokencost/blob/main/model_prices.yaml

Feel free to add your models in there. Additionally, you can update the table in the README to include those models.
